src/simple-api/src/api/xxENSONOxx.xxSTACKSxx.API.ComponentTests/Setup/Configuration.cs

```diff
@@ -0,0 +1,41 @@
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Options;
+
+namespace xxENSONOxx.xxSTACKSxx.API.ComponentTests.Setup
+{
+    public static class Configuration
```

Comment:
Do you think this class is still needed for the Component Tests?  Would the configuration not be set in memory?